default_params.config

```diff
@@ -89,6 +90,7 @@ ref_fasta_fai = "${params.ref_fasta}.fai"
 ref_fasta_pac = "${params.ref_fasta}.pac"
 ref_fasta_sa = "${params.ref_fasta}.sa"
 
+queries_multifasta = "${projectDir}/resources/regions/IS_Queries_Mtb.multi.fasta"
```

Comment:
New parameter to allow users to use a custom `IS_queries` multifasta.

Unless I'm mistaken, ideally we can add new elements by simply appending to the sequence below - could you please confirm @vrennie  and @mdediegofuertes  ? 

```
>IS6110
ATGTCCACCACCCGGGGGGAACCTCACACCTACTACGACGCCTGCGCCCGGCGCAGCCAGCCCTCCAAGC ....


> NEW_REGION
....
```

Reply:
Any comments on this assumption?

Reply:
yes sounds good